packages/story-editor/src/components/panels/design/captions/captions.js

```diff
@@ -185,67 +151,14 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
       title={__('Caption and Subtitles', 'web-stories')}
       isPersistable={!highlight}
     >
-      {isMixedValue && (
-        <Row>
-          <StyledFileInput
-            value={MULTIPLE_DISPLAY_VALUE}
-            disabled
-            aria-label={__('Filename', 'web-stories')}
-            onChange={() => handleRemoveTrack()}
-            hasMixedValue={isMixedValue}
-          />
-        </Row>
-      )}
-      {tracks &&
-        !isMixedValue &&
-        tracks.map(({ id, trackName }) => (
-          <Row key={`row-filename-${id}`}>
-            <InputRow>
-              <StyledFileInput
-                value={trackName}
-                aria-label={__('Filename', 'web-stories')}
-                onChange={() => handleRemoveTrack(id)}
-                hasMixedValue={isMixedValue}
-                disabled
-              />
-            </InputRow>
-            <Tooltip hasTail title={clearFileText}>
-              <StyledButton
-                aria-label={clearFileText}
-                type={BUTTON_TYPES.TERTIARY}
-                size={BUTTON_SIZES.SMALL}
-                variant={BUTTON_VARIANTS.SQUARE}
-                onClick={() => handleRemoveTrack(id)}
-              >
-                <Icons.Trash />
-              </StyledButton>
-            </Tooltip>
-          </Row>
-        ))}
-      {!tracks.length && (
-        <>
-          <Row expand>
-            <MediaUpload
-              onSelect={handleChangeTrack}
-              onSelectErrorMessage={__(
-                'Please choose a VTT file to use as caption.',
-                'web-stories'
-              )}
-              type={['text/vtt']}
-              title={captionText}
-              buttonInsertText={__('Select caption', 'web-stories')}
-              render={renderUploadButton}
-            />
-          </Row>
-          {uploadError && (
-            <Row expand>
-              <ErrorText role="alert">
-                {__('Upload error. Please try again', 'web-stories')}
-              </ErrorText>
-            </Row>
-          )}
-        </>
-      )}
+      <CaptionsPanelContent
+        isMixedValue={isMixedValue}
+        tracks={tracks}
+        handleChangeTrack={handleChangeTrack}
+        handleRemoveTrack={handleRemoveTrack}
+        renderUploadButton={renderUploadButton}
+        uploadError={uploadError}
```

Comment:
This should be all within `CaptionsPanelContent`, not passed as a prop.

But since this is completely unused anyway, let's just remove `uploadError` entirely.

Reply:
Fixed https://github.com/GoogleForCreators/web-stories-wp/pull/10288/commits/d2bead5d655e2e350b9bc638c3b495e1313bc4f8